about.html

```diff
@@ -0,0 +1,128 @@
+        @import url('https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap');
+        </style>
+    <title>Visit Rwanda Festivals and Events</title>
+</head>
+<body>
+    <div class="social_icon">
+        <ul>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+                      </ul>
+    </div>
+    <header id="header">
+        <div class="logo"><a href="index.html">
+          <img src="images/gorilla-logo.png" alt="logo"></a>
+        </div>
+        <nav id="nav-bar" class="nav-bar d-none">
+          <div id="close-icon" class="close-icon d-none">
+            <i class="fas fa-times-circle"></i>
+          </div>
+          <div class="nav-wrap">
+            <a class="nav-link" id="about" href="index.html" >Home</a>
+            <a class="nav-link" href="#" >Program</a>
+            <a class="nav-link" href="#" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+        </div>
+      </header>
+    <div class="banner">
+        <div class="hidden_a">
+        <div class="hidden_b">          
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>Visit Rwanda Festivals and Events</h2>
+        <p>Better known as ‘Hillywood’, the Rwanda Film Festival has been bringing films from around Rwanda, Africa and the world to Kigali every July for more than 10 years now</p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+        <h5>Please contact us per Email for any futher questions about Visit Rwanda 2021! <br> <b>mucyosofa@gmail.com</b></h5>
+            </div>
+         </div>
+    </div>
+
+    <section class="main">
+        <h2>Visit Rwanda Now</h2>
+        <div class="hr">
+         </div>
+         <div class="global">
+           <h3>Rwanda is now open. Learn more here about Virunga Nation Park.</h3>  
+         </div>
+
+         <div class="imagelogo"><img src="images/gorilla-logo.png" alt="logo"></div>
+
+         <div class="lineb"></div>
+         <div class="rda">
+             <h2>Rwanda lies just south of the Equator</h2></div> 
+             <div class="hr"></div>
+             
+         <div class="view">
+             <h6>View special offers from Rwandan tour and travel operators here.</h6>
+         </div>
+         <div class="dance">
+            <h1>2021 <br> Tradition Dance</h1> 
+            
+           </div>
+         <div class="gorila">
+             <h3>2020 <br> Nation Park</h3>  
+         </div>
```

Comment:
- [ ] On the desktop view, the bottom section images are organized in a horizontal row, not a vertical row. Please kindly fix that error.
- [ ] I can also see that the whole page has an overflow. I suggest you use `overflow-x: hidden` on the whole page and see if it works.

Reply:
- [ ] As said previously, the bottom section images are organized in a horizontal row, not a vertical row. Please kindly fix that error.